components/scream/src/physics/shoc/shoc_integ_column_stability_impl.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef SHOC_INTEG_COLUMN_STABILITY_IMPL_HPP
+#define SHOC_INTEG_COLUMN_STABILITY_IMPL_HPP
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+
+namespace scream {
+namespace shoc {
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::integ_column_stability(
+  const MemberType& team,
+  const Int& nlev,
+  const uview_1d<const Spack>& dz_zt,
+  const uview_1d<const Spack>& pres,
+  const uview_1d<const Spack>& brunt,
+  Scalar& brunt_int)
+{
+  const Int nlev_pack = ekat::pack::npack<Spack>(nlev);
+  brunt_int = 0;
+  static constexpr auto troppres = 80000;
```

Comment:
This should go into a SHOC parameters struct corresponding to the shoc.F90 comment `! Below define some parameters for SHOC`. It also needs Scalar on the LHS for BFB handling of single precision.

Reply:
Thanks. I will replace `auto` with `Scalar`. I thought about adding it to the parameter list. It is not being used at any other place in SHOC therefore, I thought about keeping it close to the only place where it is being used. I will move it to SHOC parameters file if that is more consistent or desirable.